docs/myaccount/README.md

```diff
@@ -1,152 +1 @@
 @okta/okta-auth-js/myaccount / [Exports](modules.md)
-
-# MyAccount API
```

Comment:
where this part goes?

Reply:
I'm actually not sure, all I did was run `yarn docs:myaccount` after making my changes. I didn't notice this deleted section